packages/itwinui-react/src/core/Avatar/Avatar.tsx

```diff
@@ -105,11 +112,10 @@ export const Avatar = (props: AvatarProps) => {
       style={style}
       {...rest}
     >
-      {image ?? (
-        <abbr className='iui-initials' style={{ backgroundColor }}>
-          {abbreviation?.substring(0, 2)}
-        </abbr>
-      )}
+      <abbr className='iui-initials' style={{ backgroundColor }}>
+        {image || child ? '' : abbreviation?.substring(0, 2)}
```

Comment:
what's the reason for this change? `image` will now be included twice

Reply:
Not sure what do you mean by _included twice_. I did this change since in the HTML demo, the `<abbr>` with empty text is used as the background for images. So if image or child is passed, the text is empty.

Reply:
ah nvm, it's not included twice.

still, the abbr should not be needed when there is an image. an empty abbr looks wrong to me.

i think you just want the background color, so maybe make it a div when there is an image

Reply:
I've also noticed that the default background is `white` is for some reason. This will not work well because we are also setting the svg fill to `white`.

We might want to change default background. @FlyersPh9

Reply:
> still, the abbr should not be needed when there is an image. an empty abbr looks wrong to me.
> 
> i think you just want the background color, so maybe make it a div when there is an image

To confirm, were you meaning something like [`966544a` (#1040)](https://github.com/iTwin/iTwinUI/pull/1040/commits/966544ab0346f92214f0ab11f84481eb03e5badb)?

I thought of adding a proper background like `.iui-avatar-background` instead of re-using `.iui-initials` on the `<div>` background. But I thought that might be a bigger than needed change and so went with the reuse `.iui-initials` approach.

Reply:
yup, that looks fine, although i would have done it like this for better readability:
```jsx
{image ? (
  <div className='iui-initials' style={{ backgroundColor }} />
) : (
  <abbr className='iui-initials' style={{ backgroundColor }}>
    {abbreviation?.substring(0, 2)}
  </abbr>
)}
```

alternatively we could look into moving the backgroundColor to the parent element to avoid the need for this extra div.

Reply:
> yup, that looks fine, although i would have done it like this for better readability:

I first changed it to the code to the code you showed. [`8bff798` (#1040)](https://github.com/iTwin/iTwinUI/pull/1040/commits/8bff798609c9de2dba9db457747a6d11a6127de9)

> alternatively we could look into moving the backgroundColor to the parent element to avoid the need for this extra div.

I then moved the background color to `.iui-avatar` and added a `border-radius: 50%` and also added the `.iui-initials` default background color (`background-color: hsl(72, 51%, 56%);`). [`38c3484` (#1040)](https://github.com/iTwin/iTwinUI/pull/1040/commits/38c3484cdb4476abb4e5fb96cb152e50ff86557b)

Reply:
> > I've also noticed that the default background is `white` is for some reason. This will not work well because we are also setting the svg fill to `white`.
> 
> Now `.iui-avatar` has a default `background-color: hsl(72, 51%, 56%);` (olive green)

If you look at line 85, we are setting the default value of the inline style to `white`. This is what needs to change for better support of svgs (hence why i also tagged @FlyersPh9)

Reply:
> https://github.com/iTwin/iTwinUI/pull/1040#discussion_r1095782944: I then moved the background color to `.iui-avatar` and added a `border-radius: 50%` and also added the `.iui-initials` default background color (`background-color: hsl(72, 51%, 56%);`). [`38c3484` (#1040)](https://github.com/iTwin/iTwinUI/pull/1040/commits/38c3484cdb4476abb4e5fb96cb152e50ff86557b). 

I think moving the background color from `.iui-initials` to `.iui-avatar` caused 42 test image changes. When checking on GitHub desktop, it shows no changes between the new and old images. But when checking on the GitHub website with the `GitHub PR images comparer settings` extension, it shows a few unrelated changes.

![image](https://user-images.githubusercontent.com/45748283/217017812-87a8e56e-3833-4cfa-9b7a-d8b2c73cd7ef.png)

I'm not sure which is correct. I approved these images for now and didn't revert moving `background-color` to `.iui-avatar`

[`28e04c9` (#1040)](https://github.com/iTwin/iTwinUI/pull/1040/commits/28e04c9451659abd5f7578fe0af43411865ddcd3)

Reply:
Was white set as the default background color in React incase someone used a transparent PNG as their avatar photo?  We set a background color on `img` [here in CSS](https://github.com/iTwin/iTwinUI/blob/d6fe794d2e7d4ea7a698f9d7d1b4b82337037030/packages/itwinui-css/src/avatar/avatar.scss#L69), which should fix that issue for transparent PNGs.

I think it should be safe to change from white unless I am misunderstanding why it's being set.  I'd set it to `hsl(72, 51%, 56%)` to match [what we have in CSS](https://github.com/iTwin/iTwinUI/blob/d6fe794d2e7d4ea7a698f9d7d1b4b82337037030/packages/itwinui-css/src/avatar/avatar.scss#L44).

Reply:
Lets unset it in React then. CSS will handle both cases (img and non-img).

Reply:
Agree.

Reply:
I removed the default `background-color: white` inline style in React. Now `background-color` inline style is only set when a `backgroundColor` is passed. [`ee84744` (#1040)](https://github.com/iTwin/iTwinUI/pull/1040/commits/ee847440335d571283a31c27909a15e5c1682ecf)

Just confirming before merging; this is what you meant, right? @mayank99 @FlyersPh9